backend/src/main/java/io/debezium/configserver/rest/client/JolokiaClient.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.configserver.rest.client;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.management.MalformedObjectNameException;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jolokia.client.BasicAuthenticator;
+import org.jolokia.client.J4pClient;
+import org.jolokia.client.exception.J4pException;
+import org.jolokia.client.request.J4pReadRequest;
+import org.jolokia.client.request.J4pReadResponse;
+
+@ApplicationScoped
+public class JolokiaClient {
+
+    public static final String PROPERTY_JOLOKIA_CLIENT_URI = "jolokia.client.uri";
+
+    @ConfigProperty(name = PROPERTY_JOLOKIA_CLIENT_URI)
+    private String jolokiaUrl;
+
+    @Inject
+    JolokiaAttributes jolokiaAttributes;
+
+    public List<String> getAttributeNames() {
+        return jolokiaAttributes.getAttributeNames();
+    }
+
+    public List<J4pReadResponse> getMetrics(String connectorType, String serverName, List<String> attributeNames) {
+        final J4pClient client = getJolokiaClient();
```

Comment:
Instead of having a proxy for Jolokia, why don't we access and expose the metrics directly? If Jolokia is enabled for the Debezium Connect container then the UI could use Jolokia directly, why using this proxy?

Reply:
> Instead of building this proxy for Jolokia, why don't we access and expose the metrics directly via Jolokia without an endpoint?

Are you referring to the `ConnectorURIs.LIST_CONNECTOR_METRICS_ENDPOINT` endpoint or something else?  Perhaps we can sync up on this when you're back from PTO?

Reply:
Yes, let's sync up tomorrow when I'm back.
What I meant was that we don't build that endpoint inside the backend but instead we only enable Jolokia and let the UI Frontend access it directly...? But we should think about security constraints, sure.

Reply:
I suggest we discuss this in another call and we need to reconsider how the UI frontend accesses the metrics. I think there's currently no good way than to access Jolokia directly when taking UI backend removal into account. Similarly to how we agreed on in the milestone planning to reduce effort.

Reply:
We already discussed this with the broader team on June 8th's team call and we all concluded that the current design made sense.  We can certainly rediscuss this again, but I think we need to also consider this PR is nearly 2 months old and we need to move on to other tasks too.